helm/charts/infra/templates/connector/secret.yaml

```diff
@@ -9,29 +10,32 @@ metadata:
 {{- include "connector.labels" . | nindent 4 }}
 data:
   access-key: {{ include "connector.accessKey" . | b64enc | quote }}
-{{- end }}
+{{- end }}{{/* if not accessKey */}}
 ---
 {{- if and (not .Values.connector.config.caCert) (not .Values.connector.config.caKey) }}
-{{- $secret := lookup "v1" "Secret" .Release.Namespace (printf "%s-ca" (include "connector.fullname" .)) -}}
-{{- $ca := genCA "infra-connector" 3650 -}}
+{{- $secret := lookup "v1" "Secret" .Release.Namespace (printf "%s-ca" (include "connector.fullname" .)) }}
+
 apiVersion: v1
 kind: Secret
 metadata:
   name: {{ include "connector.fullname" . }}-ca
   labels:
 {{- include "connector.labels" . | nindent 4 }}
-data:
-  ca.crt: |
+stringData:
```

Comment:
Not sure why this is being changed from `data`. It's only exchanging `b64enc` on `genCA` outputs for `b64dec` on existing secrets